board/freescale/mx6ullevk/mx6ullevk.c

```diff
@@ -581,12 +581,6 @@ static void ccgr_init(void)
 }
 
 
-int board_spi_cs_gpio(unsigned bus, unsigned cs)
-{
-        return IMX_GPIO_NR(1, 20);
-}
```

Comment:
But now the SPL build gets the empty `board_spi_cs_gpio()` stub in `drivers/spi/mxc_spi.c`. Does that work?

I would be nicer to have a single routine for both SPL and non-SPL.

Reply:
"nicer to have". I tried but was not able to get a unified routine working.